packages/forms/src/Components/BaseFileUpload.php

```diff
@@ -127,6 +131,13 @@ public function acceptedFileTypes(array | Arrayable | Closure $types): static
         return $this;
     }
 
+    public function allowReorder(bool | Closure $allowReorder = true): static
```

Comment:
Please rename this to `enableReordering()`, and the argument should be `$condition`.